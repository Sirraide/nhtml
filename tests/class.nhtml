div.class1.class2.foo.bar {
    div.inner (foobarbaz)
}
