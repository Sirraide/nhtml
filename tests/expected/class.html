<div class="bar class1 class2 foo">
    <div class="inner">foobarbaz</div>
</div>
